index.html

```diff
@@ -1,32 +1,53 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="style.css">
     <title>Awesome Book App</title>
 </head>
-<body>
-    <h1>Awesome Book App</h1>
-    <table>
-        <tbody id="book-list">
 
-        </tbody>
-    </table>
+<body>
+    <p id="date">
+        <script> document.write(new Date().toString()); </script>
+    </p>
```

Comment:
- [ ] (OPTIONAL) Good job so far 👍 kindly put the date underneath the navbar and not above so as to follow the wireframe 💯